lib/model/notes-collection.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const Request = require('./notes-schema');
+
+//Implement CRUD operations
+// Each method should invoke the proper mongoose method via your schema
+
+class DBInterface {
+  constructor(schema) {
+    this.schema = schema
+  }
+
+  get(payload) {
+    const reqParam = payload !== true ? {category: payload} : {} ;
+    return Request.find(reqParam)
+      .then(results => console.log('Showing list results', results))
+      .then(err => console.log(err));
+  }
+
+  create(payload, category) {
+    const newRequest = new this.schema({payload: payload, category: category});
+    return newRequest.save()
+      .then(results => console.log('Note Saved:', results))
+      .catch(err => console.log(err));
+  }
+
+  update()
```

Comment:
This is causing an error when I try and run your code. The update needs to be deleted or if you want to leave it in blank you can write the following: `update() {}`